asciidoc/courses/aura-dashboards/modules/2-understand-your-data/lessons/1-describe-data-model/lesson.adoc

```diff
@@ -0,0 +1,132 @@
+= Describe your movie data model
+:type: lesson
+:order: 1
+
+In this lesson, you will learn how to describe your movie data model in Neo4j Aura and design dashboards that align with stakeholder requirements. This includes understanding the structure of your project, the relationships between different components, and the properties that define them.
+
+By the end of this lesson, you should be able to:
+
+* Identify the key entities in your project
+* Describe the relationships between those entities
+* List the properties associated with each entity and relationship
+* Understand the overall structure of your graph data
+
+== Understanding requirements
+
+
+To draw a clear picture of your graph data model, consider the following questions:
+
+* What are the main nodes and properties in your data?
+* How are these nodes connected?
```

Comment:
Maybe this point could be about the properties? Relationships can have properties too.